statick_tool/plugins/discovery/cmake_discovery_plugin.py

```diff
@@ -52,6 +52,7 @@ def scan(
             "cmake",
             ".",
             "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+            "-DBUILD_GMOCK=ON",
```

Comment:
not as part of this fix, but in the future it'd be good to extract these options into the statick config file.

Reply:
I agree. I created #250 to remind me.